city_temperature_app/models.py

```diff
@@ -0,0 +1,32 @@
+from sqlalchemy import (
+    Column,
+    DateTime,
+    Float,
+    ForeignKey,
+    Integer,
+    String,
+)
+from sqlalchemy.orm import relationship
+
+from database import Base
+
+
+class DBCity(Base):

```

Comment:
Rename it to `City`. We don't use `DBCity` to indicate it's a model; in FastAPI, we differentiate using `model.City`, `schemas.City`, etc.